modules/sb-install-service-catalog.adoc

```diff
@@ -2,55 +2,48 @@
 //
 // * applications/service_brokers/installing-service-catalog.adoc
 
-[id='sb-enable-service-catalog-{context}']
-= Enabling service catalog
+[id='sb-install-service-catalog-{context}']
+= Installing service catalog
 
-Service catalog is included by default in {product-title}, but you must you must
-enable it using the procedure below.
+Service catalog is not installed by default in {product-title} 4. You can install
+it by using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
+.. Use the following command to edit the service catalog API server resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogAPIServer
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogapiservers
 ----
-.. Use the following command to create the service catalog API server.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-api-server.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog API server is now installed into the `kube-service-catalog`
-namespace.
+This causes the Operator to install the service catalog API server
+component. As of {product-title} 4, this component is installed into the
+`openshift-service-catalog-apiserver` namespace.
 
-. Create the service catalog controller manager.
-.. Create a file named `svcat-controller-manager.yaml` with the following
-contents.
+. Enable the service catalog controller manager.
+.. Use the following command to edit the service catalog controller manager resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogControllerManager
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogcontrollermanagers
 ----
-.. Use the following command to create the service catalog controller manager.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-controller-manager.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog controller manager is now installed into the
-`kube-service-catalog-controller-manager` namespace.
+This causes the Operator to install the service catalog controller
```

Comment:
I'd probably say "The Operator installs the service catalog controller componentr in the `openshift-service-catalog-controller-manager` namespace."